canvas_site_creator/static/canvas_site_creator/js/controllers/CreateNewCourseController.js

```diff
@@ -224,30 +237,37 @@
                                 ['api/course/v2/courses/']);
             var departmentId = $scope.departments[$scope.course.codeType.toLowerCase()];
             var data = {
-                departments: [departmentId],
                 registrar_code: $scope.course.code,
                 registrar_code_display: $scope.course.code,
-                school: $scope.school.id,
+                school: $scope.school.id
             };
+            // Only include departments if this is not a blueprint course
+            if(!$scope.isBlueprint) {
+                data.departments = [departmentId]
```

Comment:
missing semicolon at the end of the line?

Reply:
Good catch!